packages/vite/src/node/plugins/resolve.ts

```diff
@@ -720,8 +740,11 @@ export function tryNodeResolve(
     basedir = root
   }
 
+  const isModuleBuiltin = (id: string) =>
+    isBuiltin(options?.builtins ?? nodeLikeBuiltins, id)
```

Comment:
```suggestion
    isBuiltin(options.builtins, id)
```
`options` and `options.builtins` always exists (at least the type declares it), so I think we can simplify this. Maybe we can avoid this closure too.